toqito/matrix_props/is_right_stochastic.py

```diff
@@ -0,0 +1,74 @@
+"""Check is a matrix is right stochastic."""
+
+import numpy as np
+
+from toqito.matrix_props import is_nonnegative, is_square
+
+
+def is_right_stochastic(mat: np.ndarray) -> bool:
```

Comment:
Instead of defining two separate functions that are basically the same, why don't we just define one function:

```py
is_stochastic
```

which takes in an optional argument of either `"left"` or `"right"`? Otherwise, there's a bunch of duplicated code.